### PR TITLE
feat: urlify tooltip text

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Fixes https://github.com/snapshot-labs/snapshot/issues/3676

### Changes 
1. Added props and theme to v-tippy in SpaceProposalVotesListItem.vue

### How to test
1. Go to proposal with link in tooltip [like this](https://snapshot.org/#/arbitrumfoundation.eth/proposal/0x3be7368a662d1cf12fa4da768d626edbc013be0dc7b994fef2e24d9a54e4033a)
2. Click links

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [ ] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### CODE REVIEW

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -140,3 +140,8 @@ export async function clearStampCache(id: string, type = 'space') {
   if (type === 'avatar')
     return await fetch(`https://cdn.stamp.fyi/clear/avatar/eth:${id}`);
 }
+
+export function urlify(text: string, target = '_blank') {
+  const urlRegex = /(https?:\/\/[^\s]+)/g;
+  return text.replace(urlRegex, `<a href="$1" target="${target}">$1</a>`);
```
Comment (samuveth) [inline src/helpers/utils.ts:146]:
```suggestion
  return text.replace(urlRegex, `<a href="$1" target="${target}" rel="noopener">$1</a>`);
```

Adding noopener for security

File: src/components/SpaceProposalVotesListItem.vue [hunk 2]
```diff
@@ -114,3 +117,11 @@ const { formatCompactNumber } = useIntl();
     </div>
   </div>
 </template>
+
+<style scoped>
```
Comment (samuveth) [inline src/components/SpaceProposalVotesListItem.vue:121]: Why do we need this, should we make it global maybe?

Comment (samuveth) [reply src/components/SpaceProposalVotesListItem.vue:121]: We have some more tippy styling in style.scss

Comment (j0hnfl0w) [reply src/components/SpaceProposalVotesListItem.vue:121]: ah took it from example https://vue-tippy.netlify.app/themes#creating-a-theme

**Comment by samuveth** [comment]
Nice work @j0hnfl0w

**Pull request merged**